crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -521,6 +521,7 @@ def valid_transition_states
       "applying", "discovered", "discovering", "hardware-installed",
       "hardware-installing", "hardware-updated", "hardware-updating",
       "installed", "installing", "ready", "readying", "recovering",
+      "crowbar_upgrade",
```

Comment:
(btw, I'm not suggesting that you do the followup it as it's not that important for now)